api/src/main/java/bio/terra/tanagra/service/search/SqlVisitor.java

```diff
@@ -198,9 +198,9 @@ public String resolveTable(EntityVariable entityVariable) {
     }
 
     private String resolveTable(Table table) {
-      // projectId.datasetId.table
+      // `projectId.datasetId`.table
       return String.format(
-          "%s.%s.%s", table.dataset().projectId(), table.dataset().datasetId(), table.name());
+          "`%s.%s`.%s", table.dataset().projectId(), table.dataset().datasetId(), table.name());
```

Comment:
Hmm.. perhaps I'm wrong - but isn't it typical to include the `table` in the backticks as well?

Reply:
Maybe, but if you do, you have to include the full table qualification when the table isn't aliased i.e. ```
`my-project.dataset.foo` where foo.x =0
``` doesn't work.